internal/reminder/check_sla_close.go

```diff
@@ -20,8 +20,6 @@ func hasSLAClose(ctx context.Context, client bot.Client, logger log.Logger, inci
 				log.Trace(),
 				log.Action("do_not_notify"),
 				log.Reason("SLAHoursToClose"),
-				log.NewValue("channelID", incident.ChannelId),
```

Comment:
Suggestion: rename the parameter logger to logWriter to make it clear that it already contains the log fields.